NPM-Behind-Proxy.md

```diff
@@ -0,0 +1,40 @@
+# NPM Behind A Proxy Server
+
+### Use-cases
+
+You might need to modify `npm install` commands that accesses remote repositories ([npmjs](https://www.npmjs.com/), for example) for installing Node JS modules; if your internet access is through a [proxy server](https://en.wikipedia.org/wiki/Proxy_server).
+
+It's quite common if you are a student in college, or maybe you are a working professional; accessing the internet behind an institute/company mandated firewall.
+
+You can [locate your proxy settings](http://www.wikihow.com/Change-Proxy-Settings) from your browser's settings panel.
+
+### Using Proxy with NPM
+
+Once you have obtained the proxy settings (server URL, port, username and password); you need to configure your `npm` configurations as follows.
+
+```bash
+$ npm config set proxy http://<username>:<password>@<proxy-server-url>:<port>
+$ npm config set https-proxy http://<username>:<password>@<proxy-server-url>:<port>
+```
+You would need to replace `<username>`, `<password>`, `<proxy-server-url>`, `<port>` with the values specific to your proxy server credentials.
```

Comment:
Suggested change: I would remove 'You would need to' and have the sentence start with 'Replace'.